docs: add a table with each feature supported by model (#258)
timescale/pgai: docs/adding-embedding-integration.md

## Documentation

> Ensure that the new integration is documented.

Author: Fixed

- Document the new database function in [vectorizer-api-reference.md].
- Document any changes to the vectorizer worker in [vectorizer-worker.md].

> - Add a new row into the [Features per model table in the README file](/README.md#supported-features-per-model) indicating all the supported features.

Reviewer: ```suggestion
- Add a new row in [Supported features in each model](/README.md#supported-features-in-each-model) for your worker.
```
